src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -91,23 +92,31 @@
    "No Access" #(= "NO ACCESS" (% "accessLevel"))
    })
 
-(def ^:private realm-types ["non-TCGA" "TCGA Open Access" "TCGA Protected Access"])
+(def ^:private realm-types ["Un-published" "Published" "TCGA Open Access" "TCGA Protected Access"])
+(def ^:private realm-types-defaults [true false false false])
 (def ^:private realm-predicates
- {"non-TCGA" #(not= (config/tcga-namespace) (get-in % ["workspace" "namespace"]))
+ {"Published" #(and (not= (config/tcga-namespace) (get-in % ["workspace" "namespace"])) (get-in % ["workspace" "attributes" "library:published"]))
+  "Un-published" #(and (not= (config/tcga-namespace) (get-in % ["workspace" "namespace"])) (not (get-in % ["workspace" "attributes" "library:published"])))
```

Comment:
Also I believe we'll have to break out Published/Unpublished and TCGA Open/TCGA Controlled into separate structures (need Moran's input on this), because the logic is `(or access-predicates) AND (or realm-predicates)`. In other words, we would OR between Published and TCGA Open ... so selecting *either* of those options would show the open-access workspaces.